dashed/models.py

```diff
@@ -840,8 +840,12 @@ def latest_metadata(self):
             return
         max_time = results[0]['result']['maxTime']
         max_time = parse(max_time)
-        intervals = (max_time - timedelta(seconds=1)).isoformat() + '/'
-        intervals += (max_time + timedelta(seconds=1)).isoformat()
+        # Query segmentMetadata for 7 days back. However, due to a bug,
```

Comment:
Max has a point that some segment might be outside of the range.  -7 is a heuristic to get at least one segment.